packages/web/src/components/gcds-link/stories/gcds-link.stories.js

```diff
@@ -0,0 +1,246 @@
+export default {
+  title: 'Components/Link',
+
+  parameters: {
+    actions: {
+      argTypesRegex: '^on.*',
+      handles: ['click', 'change', 'focus', 'blur'],
+    },
+  },
+
+  argTypes: {
+    // Props
+    size: {
+      control: { type: 'radio' },
+      options: ['regular', 'small', 'inherit'],
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: 'inherit' },
+      },
+    },
+    display: {
+      control: 'select',
+      options: ['inline', 'block'],
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: 'inline' },
+      },
+    },
+    href: {
+      control: 'text',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '-' },
+      },
+    },
+    rel: {
+      control: 'text',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '-' },
+      },
+    },
+    target: {
+      control: 'text',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '_self' },
+      },
+    },
+    external: {
+      control: 'boolean',
+      table: {
+        type: { summary: 'boolean' },
+        defaultValue: { summary: false },
+      },
+    },
+    download: {
+      control: 'text',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '-' },
+      },
+    },
+    type: {
+      control: { type: 'text' },
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '-' },
+      },
+    },
+
+    // Slots
+    default: {
+      control: {
+        type: 'text',
+      },
+      table: {
+        category: 'Slots | Fentes',
+      },
+    },
+  },
+};
+
+const Template = args =>
+  `
+This is an example of
+<!-- Web component code (Angular, Vue) -->
+<gcds-link ${args.display != 'inline' ? `display="${args.display}"` : null} ${
+    args.href ? `href="${args.href}"` : null
+  } ${args.rel ? `rel="${args.rel}"` : null} ${
+    args.target != '_self' ? `target="${args.target}"` : null
+  } ${args.size != 'inherit' && args.size ? `size="${args.size}"` : null} ${
+    args.external ? `external` : null
+  } ${args.download ? `download="${args.download}"` : null} ${
+    args.type ? `type="${args.type}"` : null
+  }>${args.default}</gcds-link>
+
+<!-- React code -->
+<GcdsLink ${args.display != 'inline' ? `display="${args.display}"` : null} ${
+    args.href ? `href="${args.href}"` : null
+  } ${args.rel ? `rel="${args.rel}"` : null} ${
+    args.target != '_self' ? `target="${args.target}"` : null
+  } ${args.size != 'inherit' && args.size ? `size="${args.size}"` : null} ${
+    args.external ? `external` : null
+  } ${args.download ? `download="${args.download}"` : null} ${
+    args.type ? `type="${args.type}"` : null
+  }>${args.default}</GcdsLink>
+link.
+`.replace(/ null/g, '');
+
```

Comment:
Forgot in my earlier suggestion that the `inherit` link was supposed to be wrapped in a heading. We need to add a new template for that:

```suggestion
const TemplateSizeInherit = args =>
  `
<!-- Web component code (Angular, Vue) -->
<gcds-heading tag="h4"><gcds-link href="#">${args.default}</gcds-link></gcds-heading>

<!-- React code -->
<GcdsHeading tag="h4"><GcdsLink href="#">${args.default}</GcdsLink></GcdsHeading>
`;
```

Reply:
Yeah, absolutely! I had the idea yesterday just didn't write the code haha thanks Melanie!